### DISCUSSION TITLE
Welcome to biome!

### DISCUSSION THREAD

**Discussion by Conaclos** [biomejs/biome#2] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about
- Share ideas and feedbacks

**Comment by vighnesh153** [comment]
Why Biome? Why not continue with Rome? 

Sorry, I couldn't find any relevant discussion about this.

**Reply by Conaclos** [reply]
You can find the rationals of this choice in the [announcement blog post](https://biomejs.dev/blog/annoucing-biome).
